simulation/src/test/resources/report.md

```diff
@@ -11,8 +11,8 @@
                  drastic_slowdown[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.275952999S 	server_cpu=PT18M23.81199998S	client_received=4000/4000	server_resps=4000	codes={200=4000}
                            drastic_slowdown[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.275952999S 	server_cpu=PT18M23.81199998S	client_received=4000/4000	server_resps=4000	codes={200=4000}
         fast_400s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=82.2%	client_mean=PT0.102233333S 	server_cpu=PT10M13.4S     	client_received=6000/6000	server_resps=6000	codes={200=4934, 400=1066}
-            fast_400s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=71.0%	client_mean=PT0.091S       	server_cpu=PT9M6S         	client_received=6000/6000	server_resps=6000	codes={200=4260, 400=1740}
-                      fast_400s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=71.0%	client_mean=PT0.091S       	server_cpu=PT9M6S         	client_received=6000/6000	server_resps=6000	codes={200=4260, 400=1740}
+            fast_400s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=99.8%	client_mean=PT0.119833333S 	server_cpu=PT11M59S       	client_received=6000/6000	server_resps=6000	codes={200=5990, 400=10}
```

Comment:
@tpetracca this simulation runs with 10 clients, and the `codes={200=5990, 400=10}` bit shows that clients only perceive ten 400s in total, because all other requests happen to get routed to working nodes.